docs/enterprise-app-store/in-app-updates/index.md

```diff
@@ -674,56 +679,61 @@ namespace MAUI_IN_APP.Helpers;
-	  	int[] VersionComponents(string version)
-	  	{
-	  		return version
-	  			.Split('.')
-	  			.Select(part => int.TryParse(part, out int num) ? num : (int?)null)
-	  			.Where(num => num.HasValue)
-	  			.Select(num => num.Value)
-	  			.ToArray();
-	  	}
-	  	var currentComponents = VersionComponents(currentVersion);
-  
-	  	foreach (var app in appVersions)
-	  	{
-	  		// Convert versions to arrays of integers
-	  		var latestComponents = VersionComponents(app.Version);
-
-	  		// Compare versions component by component
-	  		for (int i = 0; i < Math.Min(currentComponents.Length, latestComponents.Length); i++)
-	  		{
-	  			var current = currentComponents[i];
-	  			var latest = latestComponents[i];
-
-	  			// You can control to update None, Beta or Live publish types you have selected on Appcircle Enterprise Store
-	  			if (latest > current && app.PublishType == (int)PublishType.Live)
-	  			{
-	  				latestAppVersion = app;
-	  				break; // Assuming once we find a valid version, we don't need to check further.
-	  			}
-	  		}
-	  	}
-  
-	  	return latestAppVersion;
-	  }
+      */
+      public static AppVersion GetLatestVersion(string currentVersion, List<AppVersion> appVersions)
+      {
+      	AppVersion latestAppVersion = null;
+
+      	// Helper function to convert version string into an array of integers
+      	int[] VersionComponents(string version)
+      	{
+      		return version
+      			.Split('.')
+      			.Select(part => int.TryParse(part, out int num) ? num : (int?)null)
+      			.Where(num => num.HasValue)
+      			.Select(num => num.Value)
+      			.ToArray();
+      	}
+      	var currentComponents = VersionComponents(currentVersion);
+
+      	foreach (var app in appVersions)
+      	{
+      		// Convert versions to arrays of integers
+      		var latestComponents = VersionComponents(app.Version);
+
+      		// Compare versions component by component
+      		for (int i = 0; i < Math.Min(currentComponents.Length, latestComponents.Length); i++)
+      		{
+      			var current = currentComponents[i];
+      			var latest = latestComponents[i];
+
+      			// You can control to update None, Beta or Live publish types you have selected on Appcircle Enterprise Store
+      			if (latest > current && app.PublishType == (int)PublishType.Live)
+      			{
+      				latestAppVersion = app;
+      				break; // Assuming once we find a valid version, we don't need to check further.
+      			}
+      		}
+      	}
+
+      	return latestAppVersion;
+      }
+
 }
 `}
-  </CodeBlock>
-  </TabItem>
+</CodeBlock>
+</TabItem>
 </Tabs>
 
+:::caution
+The code above compares major versions. For instance, if the current app version is 1.0 and the available latest version is 1.1, it **won't** be considered an update. But, if the available latest version is 2.0 in your enterprise store will be treated as an update. You can configure the logic based on your business requirements.
```

Comment:
```suggestion
The code above compares major versions. For instance, if the current app version is 1.0 and the latest available version is 1.1, it **won't** be considered an update. However, if the latest available version is 2.0, it will be treated as an update in your enterprise store. You can configure this logic based on your business requirements.
```